pkgs/applications/misc/solaar/default.nix

```diff
@@ -41,12 +43,9 @@ python3Packages.buildPythonApplication rec {
   postInstall = ''
     ln -s $out/bin/solaar $out/bin/solaar-cli
 
-    install -Dm444 -t $out/etc/udev/rules.d rules.d/*.rules
+    install -Dm444 -t $udev/etc/udev/rules.d rules.d-uinput/*.rules
```

Comment:
`rules.d-uinput/`? Either a mistake or I need to learn what that is.

Reply:
It's a path in the repo: https://github.com/pwr-Solaar/Solaar/tree/master/rules.d-uinput

Reply:
According to https://github.com/pwr-Solaar/Solaar/commit/a40b9efdcaf2c2362d46b7237b4d0b2baeb8c45d, the new udev rules contains an additional rule of `/dev/uinput`, which would hopefully fix an permission issue under wayland.